src/modules/Blur/Blur.js

```diff
@@ -1,84 +1,136 @@
 module.exports = exports = function(pixels, blur) {
-    let kernel = kernelGenerator(blur, 1), oldpix = require('lodash').cloneDeep(pixels);
-    kernel = flipKernel(kernel);
-
-    for (let i = 0; i < pixels.shape[0]; i++) {
-        for (let j = 0; j < pixels.shape[1]; j++) {
-            let neighboutPos = getNeighbouringPixelPositions([i, j]);
-            let acc = [0.0, 0.0, 0.0, 0.0];
-            for (let a = 0; a < kernel.length; a++) {
-                for (let b = 0; b < kernel.length; b++) {
-                    acc[0] += (oldpix.get(neighboutPos[a][b][0], neighboutPos[a][b][1], 0) * kernel[a][b]);
-                    acc[1] += (oldpix.get(neighboutPos[a][b][0], neighboutPos[a][b][1], 1) * kernel[a][b]);
-                    acc[2] += (oldpix.get(neighboutPos[a][b][0], neighboutPos[a][b][1], 2) * kernel[a][b]);
-                    acc[3] += (oldpix.get(neighboutPos[a][b][0], neighboutPos[a][b][1], 3) * kernel[a][b]);
-                }
-            }
-            pixels.set(i, j, 0, acc[0]);
-            pixels.set(i, j, 1, acc[1]);
-            pixels.set(i, j, 2, acc[2]);
-        }
+  let kernel = kernelGenerator(blur);
+
+  // let oldpix = require('lodash').cloneDeep(pixels);
```

Comment:
Shall we remove these sections?

Reply:
we can, but if we need to fall back to the old version at any time in the future, we can use this. They are comments so they do not affect performance. I can still remove them though.